source/codegen/common_helpers.py

```diff
@@ -1186,3 +1186,10 @@ def get_params_needing_initialization(parameters: List[dict]) -> List[dict]:
     * Outputs that are calculated/populated after the API call.
     """
     return [p for p in parameters if not (is_return_value(p) or is_get_last_error_output_param(p))]
+
+
+def get_parameter_for_error_generation(params):
```

Comment:
You'll need the `"""` docstring for this public method like the others. That's what's tripping up the validate_python part of the build, it fails when running `ni-python-styleguide` against this.

Reply:
Removed this helper method